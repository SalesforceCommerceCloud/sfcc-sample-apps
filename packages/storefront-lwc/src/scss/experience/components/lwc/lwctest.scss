.lwctest {
    color: chocolate;
    background-color: azure;
}
