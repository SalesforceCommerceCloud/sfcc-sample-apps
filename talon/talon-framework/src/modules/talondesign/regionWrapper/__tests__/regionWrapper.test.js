import { createElement } from 'lwc';
import RegionWrapper from 'talondesign/regionWrapper';

describe('talondesign/regionWrapper', () => {
    it('renders when islocked is true', async () => {
        const element = createElement('talondesign-region-wrapper', { is: RegionWrapper });

        element.componentId = "blah-blah-blah";
        element.label = "My Label!";
        element.name = "foo";
        element.isLocked = true;
        element.type = "huh";

        document.body.appendChild(element);

        expect(element).toMatchSnapshot();
    });

    it('renders when islocked is false', async () => {
        const element = createElement('talondesign-region-wrapper', { is: RegionWrapper });

        element.componentId = "blah-blah-blah";
        element.label = "My Label!";
        element.name = "foo";
        element.isLocked = false;
        element.type = "huh";

        document.body.appendChild(element);

        expect(element).toMatchSnapshot();
    });
});