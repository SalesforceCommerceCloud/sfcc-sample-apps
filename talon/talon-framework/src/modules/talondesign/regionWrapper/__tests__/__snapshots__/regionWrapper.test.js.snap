// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`talondesign/regionWrapper renders when islocked is false 1`] = `
<talondesign-region-wrapper>
  #shadow-root(open)
    <div
      class="ui-interactions-region"
      data-allow-drop="false"
      data-item-id="blah-blah-blah"
      data-region-label="My Label!"
      data-region-name="foo"
      data-region-type="huh"
    >
      <slot />
    </div>
</talondesign-region-wrapper>
`;

exports[`talondesign/regionWrapper renders when islocked is true 1`] = `
<talondesign-region-wrapper>
  #shadow-root(open)
    <div
      class="ui-interactions-region"
      data-allow-drop="true"
      data-item-id="blah-blah-blah"
      data-region-label="My Label!"
      data-region-name="foo"
      data-region-type="huh"
    >
      <slot />
    </div>
</talondesign-region-wrapper>
`;
